Update key-vault-ovw-storage-keys.md

fhokholdMSFT opened the pull request:
Update get service principal command in case search command returns too many results.

azurecla commented:
Hi __@ruslany__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ruslany). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ruslany commented:
I am Microsoft FTE.

fhokholdMSFT committed:
Update key-vault-ovw-storage-keys.md

Update get service principal command in case search command returns too many results.

BrucePerlerMS reviewed (approved):
If Frank says its right, its right. :)
